.github/workflows/post-release.yml

```
@@ -65,13 +63,14 @@ jobs:
           cluster-name: ${{ env.CLUSTER_NAME }}
           resource-group: ${{ env.CLUSTER_RESOURCE_GROUP }}
 
-      - name: '⎈ Deploy'
-        uses: 'deliverybot/helm@v1'
-        with:
-          release: envelope-game
-          namespace: ${{ env.NAMESPACE }}
-          chart: envelope-game
-          repo: https://liatrio.github.io/envelope-game
-          token: '${{ github.token }}'
-          values: |
-            image.name: ${{ env.IMAGE_NAME }}:${{ needs.docker.outputs.tag }}
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        id: install
+
+      - name: 'Helm Deploy'
+        run: |
+          helm repo add envelope-game https://liatrio.github.io/envelope-game
+          helm repo update
+          helm install envelope-game envelope-game/envelope-game -n ${{ env.NAMESPACE }}
```

as is, this will only work on the initial deployment

```suggestion
          helm upgrade --install envelope-game envelope-game/envelope-game -n ${{ env.NAMESPACE }}
```